src/vs/workbench/services/files/node/watcher/nsfw/watcherService.ts

```diff
@@ -16,15 +15,16 @@ import { IWorkspaceContextService } from 'vs/platform/workspace/common/workspace
 import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
 import { IDisposable, dispose } from 'vs/base/common/lifecycle';
 import { Schemas } from 'vs/base/common/network';
-import { isPromiseCanceledError } from 'vs/base/common/errors';
+import { filterEvent } from 'vs/base/common/event';
+import { isError } from 'util';
```

Comment:
@joaomoreno did you mean to use `errors.isError` here?

Reply:
YEAH!